queue_services/business-emailer/src/business_emailer/email_processors/__init__.py

```diff
@@ -50,7 +52,7 @@ def get_filing_info(filing_id: str) -> tuple[Filing, dict, dict, str, str]:
     am_pm = leg_tmz_effective_date.strftime("%p").lower()
     leg_tmz_effective_date = leg_tmz_effective_date.strftime(f"%B %d, %Y at {hour}:%M {am_pm} Pacific time")
 
-    return filing, business_json, leg_tmz_filing_date, leg_tmz_effective_date
+    return filing, alternate_names, business_json, leg_tmz_filing_date, leg_tmz_effective_date
```

Comment:
`get_filing_info` is a common function. you need to update all references

https://github.com/search?q=repo%3Abcgov%2Flear+path%3A%2F%5Equeue_services%5C%2Fbusiness-emailer%5C%2Fsrc%5C%2Fbusiness_emailer%5C%2F%2F+get_filing_info&type=code